pages/common/xprop.md

```diff
@@ -0,0 +1,20 @@
+# xprop
+
+> A tool for displaying window and font properties in an X server.
+> More information: <https://manned.org/xprop>.
+
+- Display the name of the root window:
+
+`xprop -root {{WM_NAME}}`
```

Comment:
I was really confusedy because I thought you have to write the window's name in `WM_NAME`. I think it would be clearer if we would remove the token syntax here. (Same for example 2 and 3)